ci/testsettings.py

```diff
@@ -3,34 +3,34 @@
 # This file is exec'd from settings.py, so it has access to and can
 # modify all the variables in settings.py.
 
-# If this file is changed in development, the development server will
-# have to be manually restarted because changes will not be noticed
-# immediately.
-
-# NOTE: setting debug = true to avoid pa11y-ci timeouts
-DEBUG = False
-
+# These settings correspond to the service container settings in the
+# .github/workflow .yml files.
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
         'NAME': 'cdhweb',
         'USER': 'root',
-        'PASSWORD': '',
+        'PASSWORD': 'cdhweb',
         'HOST': '127.0.0.1',
-        'PORT': '',
+        'PORT': '3306',
         'OPTIONS': {
             'init_command': "SET sql_mode='STRICT_TRANS_TABLES'",
-            'read_default_file': '~travis/.my.cnf'
         },
+        'TEST': {
+            'CHARSET': 'utf8',
+            'COLLATION': 'utf8_general_ci'
+        }
     },
 }
 
-# required with django 1.11 when debug is false, even for tests
-ALLOWED_HOSTS = ["*"]
+# required for database access?
+ALLOWED_HOSTS = ['*']
```

Comment:
I don't think this is required for database access, but it does take effect when debug is off. I think it would impact the test client hitting the test server in some of our tests.  Using `*` seems fine for tests, but let's revise the comment.

Reply:
For me, no tests at all would pass without this - they all raised
`RuntimeError: Database access not allowed, use the “django_db” mark, or the “db” or “transactional_db” fixtures to enable it.`
So it seems to affect everything. What should the comment say?

Reply:
That's strange... I wonder if it's because of django's test client, which maybe gets automatically initialized. The error occurred on everything, even `TestClient` tests? 

I think the comment can just say required for tests when debug is false (it isn't database specific, in spite of the errors you're seeing).